.dotnet.azure/src/Custom/Batch/AzureBatchClient.Protocol.cs

```diff
@@ -3,26 +3,41 @@
 
 using System.ClientModel;
 using System.ClientModel.Primitives;
+using System.Text.Json;
 using OpenAI.Batch;
 
 namespace Azure.AI.OpenAI.Batch;
 
 internal partial class AzureBatchClient : BatchClient
 {
-    public override async Task<ClientResult> CreateBatchAsync(BinaryContent content, RequestOptions options = null)
+    public override async Task<CreateBatchOperation> CreateBatchAsync(BinaryContent content, bool waitUntilCompleted, RequestOptions options = null)
```

Comment:
This can come later and is really an "aside" question, but I wonder if we should name this `BatchCreationOperation`.

Reply:
we need to rename everything :)